doc/nrf/ug_thingy91.rst

```diff
@@ -24,7 +24,7 @@ This guide gives you more information on the various aspects of Thingy:91.
 Connecting to Thingy:91
 ***********************
 
-For connecting to Thingy:91, you can use `LTE Link Monitor`_, `Trace Collector`_, or a serial terminal.
+For connecting to Thingy:91, you can use `LTE Link Monitor`_, :ref:`connectivity_bridge`, `Trace Collector`_ or a serial terminal.
```

Comment:
The connectivity bridge is firmware that must be loaded onto the Thingy:91, so I don't think it should be in the same list as the LTE Link Monitor and the Trace Collector, which are PC-side tools, without distinction

I might phrase this as:

```
You can connect to your Thingy:91 wirelessly (using the nRF Toolbox app) or over a serial connection (using `LTE Link Monitor`_, `Trace Collector`_, or a serial terminal)
```

Then, the "Using serial ports" section should be renamed "Using a serial terminal"


And a section named "Connecting wirelessly using nRF Toolbox" should be added below, which states:

- The Thingy:91 needs to have the Connectivity Bridge installed, which comes with the Nordic UART Service
- The Nordic UART Service needs to be enabled, see the Connectivity Bridge documentation for how to do this
- Then, the device can be connected to using the nRF Toolbox mobile app

Also, I notice we are somewhat vague in this section on the "trace collector", it doesn't have a dedicated section but is mentioned in a few spots. But that's beyond the scope of this PR I think